pkg/controllers/machinesetmigration/machineset_migration_controller.go

```diff
@@ -274,8 +274,8 @@ func (r *MachineSetMigrationReconciler) requestOldAuthoritativeResourcePaused(ct
 		// The only provider we are interested in, that reconciles the inframachinetemplate, is ibmcloud/powervs
 		// which only updates its status
 		// see: https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/controllers/ibmpowervsmachinetemplate_controller.go
-	default:
-		// Any other value is disallowed by the openAPI schema validation.
+	case machinev1beta1.MachineAuthorityMigrating:
```

Comment:
Please make sure the machine side of this change gets the same update

Reply:
Ofc, thanks for catching it. Updated.